src/extension.ts

```diff
@@ -165,6 +181,18 @@ class ExtensionManager implements vscode.Disposable {
     telemetry.logEvent('open', telemetryProperties);
   }
 
+  // Partial activation means that the CMake Tools commands are hidden
+  // from the commands pallette and the status bar is not visible.
+  // The context variable "cmake:cmtPartiallyActive" is always equal
+  // to the state setting ignoreCMakeListsMissing.
+  // To have them both always in sync, cmake:cmtPartiallyActive is set
+  // by the getter and setter of ignoreCMakeListsMissing.
+  public async partialActivation(partiallyActive: boolean) {
```

Comment:
You're right, enableExtensionFeatures reads better. What do you think of a shorter one, enableCmtFeatures? 6 letters less.

Reply:
I know it can cause double negatives, but disableCmtFeatures doesn't sound bad to me either. enableCmtFeatures would confuse me because I would wonder why an extension would not have its features enabled.
Or, enableFullFeatureSet?